src/world/area_kkj/kkj_16/AED410.c

```diff
@@ -3,9 +3,16 @@
 static char* N(exit_str_0) = "sbk_02";
 static char* N(exit_str_1) = "kkj_11";
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_16/AED410", func_80240000_AED410);
+ApiStatus func_80240000_AED410(Evt* script, s32 isInitialCall) {
+    gGameStatusPtr->peachFlags &= ~0x1;
+    gPlayerData.currentPartner = script->varTable[0];
+    return ApiStatus_DONE2;
+}
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_16/AED410", func_80240028_AED438);
+ApiStatus func_80240028_AED438(Evt* script, s32 isInitialCall) {
+    gPlayerStatus.unk_0D = evt_get_float_variable(script, *script->ptrReadPos);
```

Comment:
args temp pleez (`Bytecode* args = script->ptrReadPos;` and then later `*args++`)